app/javascript/modules/listingDetailsAside/ListingDetailsProcess.tsx

```diff
@@ -78,7 +79,7 @@ export const ListingDetailsProcess = ({
         listing.Leasing_Agent_Phone ||
         listing.Office_Hours ||
         listing.Leasing_Agent_Title) &&
-        isListingRental && (
+        (isListingRental || !seeTheUnitEnabled) && (
```

Comment:
Should that be `(isListingRental || (isListingSale && !seeTheUnitEnabled))`?  The listings Kat mentioned are sales, so it seems a little strange to say "if rental or sales", since wouldn't that be everything?

Reply:
I think that is functionally the same thing

Reply:
I think that is functionally the same thing because isListingRental === !isListingSale

Reply:
Are there sales units where `seeTheUnitEnabled` is false?